src/components/components/footer/footer.js

```diff
@@ -132,7 +132,7 @@ const footerNavigationList1 = [
   },
   {
     title: 'Affiliations',
-    target: '/affiliations'
+    target: '/organizations'
```

Comment:
This is confusing I think, shouldn't the `/affiliations` change to point where the `/organizations` is pointing now?

Reply:
Yes, but it's P2